### PR TITLE
Fix for yaml front matter on Windows machines.

### LINKED ISSUE #5: Fails with windows line endings
It expects '\n\n', and therefore the splice fails if the line endings are windows style '\r\n\r\n'

### PR DESCRIPTION
**Pull request by avishnyak**
Fixes #5 
Fixes formatting

### CODE REVIEW

File: test/test.mocha.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ describe('gulp-markdown-docs', function() {
         });
     });
 
-    it('should parse one file', function(done) {
+    it.only('should parse one file', function(done) {
```
Comment (sojournerc) [inline test/test.mocha.js]: remove only

**Review by sojournerc — COMMENTED** [review]
Thanks for the PR! - I hope I'm not missing anything amongst the formatting changes. 

Do all tests pass after you remove the `.only`?

File: test/test.mocha.js (hunk 1 above)
Comment (avishnyak) [reply test/test.mocha.js]: Will do.

**Comment by avishnyak** [comment]
The tests didn't pass for me even before I started so I thought they needed to be fixed (which is why I forgot to remove the ```.only```).

**Comment by avishnyak** [comment]
@sojournerc I compared the fixtures to the data generated by the mocha tests.  

- There were a number of changes related to an updated CSS from highlight.js which I fixed by modifying the fixtures.
- There was 1 bug related to the meta attributes expecting an undefined where it was actually an empty object from the lodash function - I fixed this.
- The tests still don't pass on my machine but manually diffing the results of the mocha test to the fixtures shows 0 diffs.  I suspect this is a line ending situation in the compare since it's not a real diff that knows how to deal with whitespace and line ending differences.

**Comment by sojournerc** [comment]
Admittedly the tests are a bit fragile. I've pulled your branch and updated the tests (to work with unix line endings). Everything looks good.

I'm going to bump the major version as this adds the new library which could have unexpected consequences out in the wild. New version is `1.0.0`

Not sure what the best approach is to have test fixtures work with all platforms. I'm certainly open to suggestions.

**Pull request merged**